Source/SVGKImage.h

```diff
@@ -290,6 +290,9 @@ typedef void (^SVGKImageAsynchronousLoadingDelegate)(SVGKImage* loadedImage, SVG
 /*! returns all the individual CALayer's in the full layer tree, indexed by the SVG identifier of the SVG node that created that layer */
 - (NSDictionary*) dictionaryOfLayers;
 
+/*! returns all the individual CALayer's in the full layer tree as an array */
+- (NSArray*) arrayOfLayers;
```

Comment:
Thought you might not like that :) To be honest, the reason I've added it is because dictionaryOfLayers does not work at the moment, and I wasn't able to fix it right off.
Will submit a new pull request when I do.